.github/workflows/ci.yml

```diff
@@ -81,11 +81,14 @@ jobs:
   #       python -c "from urllib.request import urlretrieve; urlretrieve('${{ env.BUILDER_HOST }}/${{ env.BUILDER_SOURCE }}/${{ env.BUILDER_VERSION }}/builder.pyz?run=${{ env.RUN }}', 'builder.pyz')"
   #       python builder.pyz build -p ${{ env.PACKAGE_NAME }} downstream
 
-  # osx:
-  #   runs-on: macos-latest
-  #   steps:
-  #   - name: Build ${{ env.PACKAGE_NAME }} + consumers
-  #     run: |
-  #       python3 -c "from urllib.request import urlretrieve; urlretrieve('${{ env.BUILDER_HOST }}/${{ env.BUILDER_SOURCE }}/${{ env.BUILDER_VERSION }}/builder.pyz?run=${{ env.RUN }}', 'builder')"
-  #       chmod a+x builder
-  #       ./builder build -p ${{ env.PACKAGE_NAME }} --spec=downstream
+  macos:
+    runs-on: macos-${{ matrix.version }}
+    strategy:
+      matrix:
+        version: [10.15]
```

Comment:
Can we add 11? Usually MacOs is quickly adopted and 11 was around for half year or more already.

Reply:
No we cannot. See the commit where It took it out, because GitHub Actions fails it immediately, before even getting to our script.

Reply:
We can add it back when they fix it.